.github/workflows/run_unit_tests.yml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest]
```

ubuntu-22.04 doesn't have old Python 3.8.x patches
```suggestion
        os: [windows-latest, ubuntu-20.04]
```